core/frontend/src/extension/ExtensionRuntime.ts

```diff
@@ -7,92 +7,473 @@
  */
 
 /* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable @itwin/no-internal-barrel-imports */
```

Comment:
i think we can remove this eslint disable line

Reply:
I think we need it because we are importing from `../core-frontend`?

Reply:
oh sorry missed that, youre right, ignore this comment